src/screens/repo/screens/build/menu.js

```diff
@@ -58,31 +58,28 @@ export default class BuildMenu extends Component {
 
 		const hideCancel = assertBuildMatrix(build) && !proc;
 
+		const rightSide = !build ? (
+			undefined
+		) : (
+			<section>
+				{build.status === "peding" ||
```

Comment:
Typo? Should be pending.

Reply:
I think so, but it was in the master
https://github.com/drone/drone-ui/blob/master/src/screens/repo/screens/build/menu.js#L69

 In any case I fixed in last commit